package.json

```diff
@@ -24,6 +24,7 @@
     "@babel/preset-env": "^7.14.5",
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.14.5",
+    "@department-of-veterans-affairs/component-library": "^2.5.7",
```

Comment:
To be updated when https://github.com/department-of-veterans-affairs/component-library/pull/161/ is merged.